docs/eternalcore/using-API.md

```diff
@@ -0,0 +1,105 @@
+# EternalCore Developer API
+
+As part of open-source community, we support developers who want to create their own plugins using our API. We provide a simple and easy to use API that allows you to create your own plugins using our API.
+
+## 📚 Dependency Management
+
+To use our work in your plugin, You need to install correct artifact for your project. Our plugin supports Maven, Gradle Kotlin, Gradle Groovy and Gradle Groovy and SBT.
+To use latest release check [maven repository](https://repo.eternalcode.pl/#/releases/com/eternalcode/eternalcore-api).
+
+### Add repository:
+
+For Gradle:
+```groovy
+maven {
+    url = uri("https://repo.eternalcode.pl/releases")
+}
+```
+
+For Maven:
+```xml
+<repository>
+    <id>eternalcode-reposilite-releases</id>
+    <name>EternalCode Repository</name>
+    <url>https://repo.eternalcode.pl/releases</url>
+</repository>
+```
```

Comment:
I suggest using code groups, and please add gradle kotlin syntax. 

https://vitepress.dev/guide/markdown#code-groups
kotlin:
```kts
maven {
    url = uri("https://repo.eternalcode.pl/releases")
}
```

groovy:

```groovy

maven {
    url "https://repo.eternalcode.pl/releases"
}

```

Reply:
`maven("https://repo.eternalcode.pl/releases")`
should work for kts and groovy